.github/workflows/test_ubuntu_python.yml

```diff
@@ -23,10 +23,11 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DISPLAY: ':99.0'
+      SETUPTOOLS_ENABLE_FEATURES: 'legacy-editable'
```

Comment:
`SETUPTOOLS_ENABLE_FEATURES:  'legacy-editable'` is not needed anymore. The traceback you're referring to is likely outdated.

```suggestion
```